contrib/sidechain-manipulation.py

```diff
@@ -118,7 +118,26 @@ def __init__(self, value):
 		print("Sending %s to %s..." % (sys.argv[3], send_address))
 		print("(nonce: %s)" % nonce)
 
-		txid = bitcoin.sendtoaddress(send_address, Decimal(sys.argv[3]))
+		try:
+			tx_hex = bitcoin.createrawtransaction([], {send_address: Decimal(sys.argv[3])})
+			tx_hex = bitcoin.fundrawtransaction(tx_hex)['hex']
+			tx = bitcoin.signrawtransaction(tx_hex)
+			assert(tx['complete'])
+
+			tx_hex = tx['hex']
+
+			total_length  = len(tx_hex)/2
+			total_length += 14*32
+			total_length += 1000 # wild guess
```

Comment:
this was calculated by adding together full_contract, scriptSecond
then i added an entirely arbitrary amount to that